providers/build.info

```diff
@@ -1 +1,15 @@
 SUBDIRS=common default
+
+IF[{- !$disabled{fips} -}]
+  SUBDIRS=fips
+  MODULES=fips
+  IF[{- defined $target{shared_defflag} -}]
+    SHARED_SOURCE[fips]=fips.ld
```

Comment:
Change `SHARED_SOURCE` to `SOURCE`.  `SHARED_SOURCE` is only meant for libraries, where some sources are only used for the shared variant.

Reply:
Fixed